Samples/Deploy-AzureDataFactory/Deploy-AzureDataFactory.ps1

```diff
@@ -0,0 +1,828 @@
+<#
+
+.SYNOPSIS
+
+Deploys an Azure Data Factory (ADF) from the command line. 
+
+
+.DESCRIPTION
+
+The script only deploys linked services, datasets and pipelines for a data factory, no current suport for Gateways.	
+For this script to run correctly, in the PowerShell context in which this is called, you should have already loged in
+to your Azure account and selected the correct subscription, e.g.:
+	Login-AzureRmAccount
+	Get-AzureRmSubscription �SubscriptionName �YourSubscription� | Select-AzureRmSubscription
+
+The script expects the ADF's components to have specific suffixes for each of the elements
+(linked service, dataset, pipeline) in order for them to be found and deployed correctly. 
+The suffixes are configurable through the *FilenameSuffix set of parameters.
+
+Support for linked services connection strings overwriting is supported as well, 
+as long as that is passed in through the DataFactoryConfigFile parameter.
+
+.PARAMETER SetupMode 
+
+The mode in which the script will run. Valid options are:
+	CHECK                         # Checks Azure Data Factory existence
+	INSTALL                       # Deploys the new Azure Data Factory
```

Comment:
We should not use the terminology "install" or "uninstall" anywhere. This should either be create/delete or deploy/remove, respectively; this is consistent with the terminology we use in the product and also Azure-wide.